### PR TITLE
chore(operator): add openebs-operator 2.4.0

### PR DESCRIPTION
**Pull request by akhilerm**
- add cstor operator 2.4.0
- add openebs operator 2.4.0
- add ndm operator 1.0.1
- add zfs operator 1.2.0
- add openebs operator arm & lite version

NOTE : From release 2.5.0 a separate arm version operator file will not be maintained in favour of the multiarch image support.

Signed-off-by: Akhil Mohan <akhil.mohan@mayadata.io>

### CODE REVIEW

**Review by pawanpraka1 — COMMENTED** [review]
zfs-operator yaml looks fine.

**Comment by pawanpraka1** [comment]
@akhilerm can we update the zfs-operator.yaml with 1.2.1 release tag. We just released a patch fix.

**Comment by akhilerm** [comment]
Done @pawanpraka1

File: 2.4.0/zfs-operator.yaml [hunk 1]
```diff
@@ -1785,14 +1785,14 @@ spec:
...
-        openebs.io/version: 1.2.0
+        openebs.io/version: 1.2.1
     spec:
       priorityClassName: system-node-critical
       serviceAccount: openebs-zfs-node-sa
       hostNetwork: true
       containers:
         - name: csi-node-driver-registrar
-          image: quay.io/k8scsi/csi-node-driver-registrar:v1.2.0
+          image: quay.io/k8scsi/csi-node-driver-registrar:v1.2.1
```
Comment (pawanpraka1) [inline 2.4.0/zfs-operator.yaml:1795]: revert this.

File: zfs-operator.yaml [hunk 2]
```diff
@@ -1785,14 +1785,14 @@ spec:
...
-        openebs.io/version: 1.2.0
+        openebs.io/version: 1.2.1
     spec:
       priorityClassName: system-node-critical
       serviceAccount: openebs-zfs-node-sa
       hostNetwork: true
       containers:
         - name: csi-node-driver-registrar
-          image: quay.io/k8scsi/csi-node-driver-registrar:v1.2.0
+          image: quay.io/k8scsi/csi-node-driver-registrar:v1.2.1
```
Comment (pawanpraka1) [inline zfs-operator.yaml:1795]: revert this one also.

Comment (akhilerm) [reply zfs-operator.yaml]: Done.

File: 2.4.0/zfs-operator.yaml (hunk 1 above)
Comment (akhilerm) [reply 2.4.0/zfs-operator.yaml]: Done.

**Review by pawanpraka1 — APPROVED** [review]
zfs-operator.yaml looks good.

**Review by shubham14bajpai — APPROVED** [review]
lgtm

File: ndm-operator.yaml [hunk 3]
```diff
@@ -169,7 +169,7 @@ metadata:
   labels:
     name: openebs-ndm-operator
     openebs.io/component-name: ndm-operator
-    openebs.io/version: 2.3.0
+    openebs.io/version: 2.4.0-RC1
```
Comment (kmova) [inline ndm-operator.yaml:172]: @akhilerm -- should this be 2.4.0?

Comment (akhilerm) [reply ndm-operator.yaml:172]: Done.

**Review by kmova — APPROVED** [review]

**Pull request merged**